src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf

```diff
@@ -322,6 +322,18 @@
                 <source>This is not a valid UUID.</source>
                 <target>Dëst ass keng gëlteg UUID.</target>
             </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Dëse Wäert sollt e puer vun  {{ compared_value }} sinn.</target>
```

Comment:
There are two white spaces after `puer vun`. Let's remove one of them.

Reply:
fixed it thank you!